### ISSUE TITLE
Crash during Initialization MCv1.19.2

### ISSUE THREAD

**Issue by Happenstance-14** [Ladysnake/Effective#259]
Client crashes during startup.  Occurs even when only Effective & QFAPI/QSL are installed — so probably something internal — but only once the instance is well-worn with a ton of other mods IIRC (not sure how that works..).  Crashlog attached below.

[crash-2024-06-09_17.09.19-client.txt](https://github.com/user-attachments/files/15755189/crash-2024-06-09_17.09.19-client.txt)

Running Windows 11 on Lenovo;  AMD Ryzen 7 5825U w/ Radeon Graphics

- Quilt 0.26.0
- QFAPI 4.0.0-beta.30 + QSL 3.0.0-beta.29 w/ FAPI 0.77.0
- Effective 2.1.1

Available to respond.

**Comment by UpcraftLP** [comment]
Known issue, downgrade qfapi to `4.0.0-beta.30+0.76.0-1.19.2`

duplicate of #250
